cms/djangoapps/contentstore/views/item.py

```diff
@@ -1242,8 +1247,22 @@ def create_xblock_info(xblock, data=None, metadata=None, include_ancestor_info=F
                 else:
                     show_review_rules = True
 
+                # If this block is not currently a proctored exam,
+                # the best way for us to tell whether it was was *ever* configured
+                # as a proctored exam is by checking whether there is an exam record
+                # associated with the block's ID.
+                # If an exception is not raised, then we know that such a record
+                # exists, indicating that this *was* once a proctored exam.
+                was_ever_proctored_exam = True
```

Comment:
#NIT, I feel like it would read better to wrap this logical block (line 1250 - 1261) into a helper function

Reply:
Good point. I pushed a new commit.